packages/website/ts/blockchain.ts

```diff
@@ -543,6 +554,13 @@ export class Blockchain {
 
         this._dispatcher.updateBlockchainIsLoaded(true);
     }
+    public async getFirstAccountIfExistsAsync() {
```

Comment:
This looks like it could be useful inside of web3 wrapper itself. Further down this pattern is used:
```typescript
const addresses = await this._web3Wrapper.getAvailableAddressesAsync()
const userAddressIfExists = addresses[0] || '';
```
Since this isn't a UI layer, how do you feel about not returning an empty string but null or undefined?

Reply:
Once we made this method return undefined when no addresses are found, it was no longer needed. Now we simply do:

```
const userAddresses = await this._web3Wrapper.getAvailableAddressesAsync();
this._userAddressIfExists = userAddresses[0];
```